server/src/main/java/org/opensearch/indices/IndicesService.java

```diff
@@ -215,18 +215,56 @@ public class IndicesService extends AbstractLifecycleComponent
     private static final Logger logger = LogManager.getLogger(IndicesService.class);
 
     public static final String INDICES_SHARDS_CLOSED_TIMEOUT = "indices.shards_closed_timeout";
+    public static final String INDICES_REQUEST_CACHE_DISK_CLEAN_THRESHOLD_SETTING_KEY = "cluster.request_cache.disk.cleanup_threshold_percentage";
```

Comment:
Why does this one start with "cluster" instead of "indices" like the rest?

Reply:
we want this to be a cluster level setting.